Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -6997,6 +7065,14 @@ class X86Assembler {
             vexNdsLigWigTwoByteOp(simdPrefix, opcode, dest, a, b);
         }
 
+        void vexNdsLigWigCommutativeTwoByteOp(OneByteOpcodeID simdPrefix, TwoByteOpcodeID opcode, RegisterID dest, RegisterID a, RegisterID base, int offset)
+        {
+            // Since this is a commutative operation, we can try switching the arguments.
+            if (regRequiresRex(base))
+                std::swap(a, base);
+            vexNdsLigWigTwoByteOp(simdPrefix, opcode, dest, a, base, offset);
+        }
+
```

Comment:
This is wrong since swapping `a` and `base` causes completely different meaning.
Commutative cannot be defined for instructions one side is memory and one side is register.